nf_wochenende.nf

```diff
@@ -133,7 +133,7 @@ workflow {
     // multiqc
     //multiqc(bam_stats.out.collect(), bam_stats.out)
 
-    // metagen window filter
+    // metagen window filter - @Lisa - I think this step is covered in plots and or growth rate so might not be needed?
```

Comment:
As far as I can see, this exact stept, which filters out human or mouse reads and creates the `.filt.sort.csv`, doesn't seem to be covered by plots yet

Reply:
Oh, you added that part... then we can of course get rid of this extra process

Reply:
ok, pls remove